masonite/providers/BroadcastProvider.py

```diff
@@ -14,7 +14,7 @@ def register(self):
         self.app.bind('BroadcastConfig', broadcast)
         self.app.bind('BroadcastPusherDriver', BroadcastPusherDriver)
         self.app.bind('BroadcastAblyDriver', BroadcastAblyDriver)
-        self.app.bind('BroadcastManager', BroadcastManager(self.app))
+        self.app.bind('BroadcastManager', BroadcastManager)
```

Comment:
So I think we should still instantiate the drivers the `self.app` but inside the contractor we can get rid of creating the driver. If that makes sense. Let me know if you have anymore questions.